src/main/resources/hudson/scm/listtagsparameter/ListSubversionTagsParameterDefinition/help-tagsDir.html

```diff
@@ -38,6 +38,6 @@
   When you enter the URL, Jenkins automatically checks if it can connect to it.
   If access requires authentication, you'll be prompted for the necessary
   credential. If you already have a working credential but would like to change
-  it for some other reasons, click <a href="{0}/scm/SubversionSCM/enterCredential" target="_blank">
-  this link</a> and specify a different credential.
+  it for some other reasons, you can <a href="{0}/credential" target="_blank"
```

Comment:
I suspect you meant

```suggestion
  it for some other reasons, you can <a href="{0}/credentials" target="_blank"
```

Did you actually try it?

Reply:
Sigh, thanks. I clearly meant to add the `s` as indicated by the commit message, I probably lost track of that when I actually made the change (I should have use copy+paste).